frontend/projects/ui/src/app/services/marketplace.service.ts

```diff
@@ -93,6 +108,23 @@ export class MarketplaceService implements AbstractMarketplaceService {
     shareReplay(1),
   )
 
+  private readonly filteredMarketplace$ = combineLatest([
```

Comment:
Maybe a comment or something would be helpful, so later it is clear what we filter here at a glance?